vanilla-framework/vanilla-framework
# Update the floats utility and add breakpoint support

## Done
Updated the floats utility to the new naming conventions. Added breakpoint support. Deprecated the old float classes.

## QA
- Pull down branch and run `./run serve`
- Go to http://0.0.0.0:8101/vanilla-framework/
- Check all the utility float examples work
- Check the documentation is good

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/772
